src/graphql/data/nft/NftBalance.ts

```diff
@@ -161,14 +127,19 @@ export function useNftBalanceQuery(
       last,
       before,
     },
-    { fetchPolicy: 'store-or-network' }
-  )
-  const { data, hasNext, loadNext, isLoadingNext } = usePaginationFragment<NftBalancePaginationQuery, any>(
-    nftBalancePaginationQuery,
-    queryData
-  )
-  const walletAssets: WalletAsset[] = data.nftBalances?.edges?.map((queryAsset: NftBalanceQueryAsset) => {
-    const asset = queryAsset.node.ownedAsset
+    fetchPolicy: 'cache-and-network',
+  })
+
+  const hasNext = data?.nftBalances?.pageInfo?.hasNextPage
+  const loadMore = () =>
+    fetchMore({
```

Comment:
does this need an argument on how many to `fetch`? Believe backend has a default but might be good to explicity define the number of items in subsequent requests

Reply:
`fetchMore` will automatically use the previous params unless overwritten https://www.apollographql.com/docs/react/pagination/core-api/